input/fsh/SD_Genomics.fsh

```diff
@@ -235,20 +243,24 @@ Description:    "The area of the genome region referenced in testing for variant
 * component[genomicReferenceSequenceId] ^definition = "Range(s) of DNA sequence examined. The genomic reference sequence is a contiguous stretch of chromosome DNA that spans all of the exons of the gene and includes transcribed and non transcribed stretches. For this ID use either the NCBI genomic nucleotide RefSeq IDs with their version number (see: NCBI.NLM.NIH.Gov/RefSeq) or use the LRG identifiers, without transcript (t or p) extensions -- when they become available. (source: LOINC)"
 * component[genomicReferenceSequenceId].code = LNC#48013-7
 
-// * insert ReduceText
-// * insert ReduceText(referenceRange)
-// * insert ReduceText(component)
-// These are the additional MS on top of US Core Lab Observation
 * component and component.code and component.value[x] and component.dataAbsentReason MS
 
-
-//* identifier ^slicing.discriminator.type = #value
-//* identifier ^slicing.discriminator.path = "type.coding.code"
-//* identifier ^slicing.rules = #open
-//* identifier contains
-//    AccessionIdentifier 0..1 MS and
-//    FillerOrderNumber 0..1 MS and
-//    PlacerOrderNumber 0..1 MS
-//* identifier[AccessionIdentifier].type = IDTYPE#ACSN
-//* identifier[FillerOrderNumber].type = IDTYPE#FILL
-//* identifier[PlacerOrderNumber].type = IDTYPE#PLAC
\ No newline at end of file
+/* 
+Profile:    GenomicDiagnosticImplication
```

Comment:
I thought you were eliminating this profile? Did I get this wrong?

Reply:
This was commented out and doesn't show in the IG. Do you want it completely deleted?

Reply:
The snippet in the comparison view didn't show the comment. No problem, my oversight.